reusable_workflows/shared/messages.py

```diff
@@ -1,10 +1,11 @@
 # Messages
 CLOSING_MESSAGE = """Dear @{},
 
-This repository does not accept external contributions yet.
+Thank you for contributing! Unfortunately this repository does not accept external contributions yet.
 
-We are therefore closing this Pull Request, thank you for your understanding.
+We are working on enabling this by aligning our internal processes and our CI setup to handle external contributions. However this will take some time to set up so in the mean time we unfortunately have to close this Pull Request.
```

Comment:
```suggestion
We are working on enabling this by aligning our internal processes and our CI setup to handle external contributions. However this will take some time to set up so in the meantime we unfortunately have to close this Pull Request.
```